src/components/link/index.njk

```diff
@@ -59,35 +56,35 @@ Link component, with four variants:
     'rows' : [
       [
         {
-          text: 'href'
+          text: 'classes'
         },
         {
           text: 'string'
         },
         {
-          text:'Yes'
+          text:'No'
         },
         {
-          text: 'The value of the link href attribute'
+          text: 'The available modifiers for the link type: --back, --muted, --download and --skip'
```

Comment:
This needs updating as they need to pass the entire class name e.g. `govuk-c-link--back`

Reply:
Yes, it was the same before (need to pass the entire class), but for some reason only the modifiers were mentioned. I'll update to specify the entire class.